repos/clicommandlineparser.proj

```diff
@@ -2,7 +2,9 @@
 <Project ToolsVersion="15.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))/dir.props" />
   <PropertyGroup>
-    <BuildCommandArgs>--restore --pack --configuration $(Configuration)</BuildCommandArgs>
+    <!-- In Windows -restore is already passed by default by build.cmd to build.ps1 -->
```

Comment:
@safern can you separate these non-core changes into another PR? Also I suspect we should just file issues for them so they are tracked.

Reply:
Ok, I will delete them from this PR.